src/sdk/formatting/formatCurrency.ts

```diff
@@ -0,0 +1,7 @@
+import { formatNumber } from './formatNumber';
+
+export const formatCurrency = (number: number, locale: string = 'en', currency: string = 'USD'): string =>
+	formatNumber(number, null, new Intl.NumberFormat(locale, {
+		style: 'currency',
+		currency
+	}));
```

Comment:
Can you also add support for `wholeUnitsOnly` (or a better name if you can think of one)
```ts
export const formatCurrency = (number: number, locale: string = 'en', currency: string = 'USD', wholeUnitsOnly: boolean = false): string =>
	formatNumber(number, null, new Intl.NumberFormat(locale, {
		style: 'currency',
		currency,
		maximumFractionDigits: wholeUnitsOnly ? 0 : undefined
	}));
```